api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -275,6 +299,17 @@ private void assertPaymentCycleIsFullyPaid(PaymentCycle paymentCycle, List<Local
         assertThat(payment.getPaymentAmountInPence()).isEqualTo(expectedVoucherEntitlement.getTotalVoucherValueInPence());
     }
 
+    private void assertPaymentCycleHasNoPayment(PaymentCycle paymentCycle, PaymentCycleVoucherEntitlement expectedVoucherEntitlement) {
+        assertThat(paymentCycle.getCycleStartDate()).isEqualTo(LocalDate.now());
+        assertThat(paymentCycle.getCycleEndDate()).isEqualTo(LocalDate.now().plusDays(27));
+        assertThat(paymentCycle.getChildrenDob()).isEmpty();
+        assertThat(paymentCycle.getVoucherEntitlement()).isEqualTo(expectedVoucherEntitlement);
+        assertThat(paymentCycle.getPaymentCycleStatus()).isEqualTo(PaymentCycleStatus.INELIGIBLE);
+        assertThat(paymentCycle.getCardBalanceInPence()).isNull();
```

Comment:
This isn't set unless a payment is to be made, this doesn't feel correct, but its what the system currently does.

Reply:
Until we have made a balance check we cannot populate `CardBalanceInPence`. As we're not making a payment, we don't make a balance check. What other value could `CardBalanceInPence` have?

Reply:
I think the point was that I thought we might always want to know what the balance was when a new PaymentCycle was created, rather than only retrieve it if we need to make a payment. Feels like it might be useful for MI purposes if nothing else.